tcms/testruns/tests/test_views.py

```diff
@@ -824,3 +826,25 @@ def test_should_fail_when_try_to_change_status_without_permissions(self):
         self.assertRedirects(
             self.client.get(self.url, {'finished': 1}),
             reverse('tcms-login') + '?next=%s?finished=1' % self.url)
+
+
+class TestCreateTestUsers(AssignPermissionsToTestUser):
+
+    def test_with_change_status_to_finished(self):
+        initiate_user_with_default_setups(self.tester)
+        self.test_run = TestRunFactory(product_version=self.version,
+                                       plan=self.plan,
+                                       build=self.build,
+                                       manager=self.tester,
+                                       default_tester=self.tester)
+
+        self.url = reverse('testruns-change_status', args=[self.test_run.pk])
+
+        super().remove_all_permissions_except_tested("testruns.change_testrun")
```

Comment:
1. there is no need to call `super().`, you can call `self.remove_all_permissions_except_tested("testruns.change_testrun")` because you inherit all the methods from your parent class

2.
```python
self.tester.user_permissions.add(*Permission.objects.all())
print("count before:", self.tester.user_permissions.count())

super().remove_all_permissions_except_tested("testruns.change_testrun")

print("count after:", self.tester.user_permissions.count())
```
produced this:
```
$ ./manage.py test tcms.testruns.tests.test_views.TestCreateTestUsers
> ...
> count before: 166
> count after: 1
> ...
```
so it looks like this is working as expected.